src/components/hero/SkillsGroup.tsx

```diff
@@ -0,0 +1,24 @@
+import { FunctionComponent } from "react";
+interface ISkillsGroupProps {
+  title: string;
+  skills: string[];
+}
+const SkillsGroup: FunctionComponent<ISkillsGroupProps> = ({
+  title,
+  skills,
+}) => {
+  return (
+    <div>
+      <h2>{title}</h2>
+      <ul className="ball">

```

Comment:
We can use more descriptive class name `<ul className="ball">` => **`<ul className="skills-list">`**

Reply:
- [ ] Done [04d1f5f](https://github.com/AnastasiiaSorina/Portfolio-Project/pull/7/commits/04d1f5f7e16e9493e0d94389678547fa32833269)